spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/controller/TaskExecutionController.java

```diff
@@ -97,7 +97,7 @@ public TaskExecutionController(TaskExplorer explorer, TaskService taskService,
 	@ResponseStatus(HttpStatus.OK)
 	public PagedResources<TaskExecutionResource> list(Pageable pageable,
 			PagedResourcesAssembler<TaskJobExecutionRel> assembler) {
-		Page<TaskExecution> taskExecutions = explorer.findAll(pageable);
+		Page<TaskExecution> taskExecutions = this.taskService.findAll(pageable);
```

Comment:
Any specific reason why this can't the `masking` be handled within this method instead of creating those methods in `TaskExplorer` into `TaskService` again?